### PR TITLE
Add support for parse_mode in editMessageMedia and answerInlineQuery 

### PR DESCRIPTION
**Pull request by SpEcHiDe**
mb, related to #7

### CODE REVIEW

File: src/transformer.ts [hunk 1]
```diff
@@ -26,9 +26,37 @@ const buildTransformer = (parseMode: string) => {
...
+            !("parse_mode" in result.input_message_content)
+          ) {
+            newPayload.results[resulto].input_message_content.parse_mode = normalisedParseMode;
+          }
+          else if (!("parse_mode" in result)) {
+            newPayload.results[resulto].parse_mode = normalisedParseMode;
+          }
+        }
+      }
+    }
```
Comment (KnightNiwrem) [inline src/transformer.ts:55]: There doesn't appear to be much similarities in logic between editMessageMedia and answerInlineQuery. We might benefit from separating how each case is handle with a switch statement instead.

```ts
switch(method) {
  case 'editMessageMedia':
    // ...
    break;
  case 'answerInlineQuery':
    // ...
    break;
  default:
    // ...
}
```

While changing it to the above form, we might want to consider excluding `"parse_mode" in payload` from the initial check and shifting that into the switch statement's default case instead.

Also consider [for...of](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of) if you don't need the index of the array.

**Review by KnightNiwrem — CHANGES_REQUESTED** [review]

File: src/transformer.ts (hunk 1 above)
Comment (SpEcHiDe) [reply src/transformer.ts]: I think index is required to update the payload, don't know if can use spreads, and avoid the loop.

Comment (KnightNiwrem) [reply src/transformer.ts]: For...of is a loop syntax, not spread.

Comment (KnightNiwrem) [reply src/transformer.ts]: If your concern is with the array order for the new object, there are things like `push`, taking advantage of the fact that the iteration is done in order.

Alternatively, `map` to operate on the entire input array to assign to the new object.

Comment (SpEcHiDe) [reply src/transformer.ts]: Ok, is the updated changes correct ?

**Review by KnightNiwrem — APPROVED** [review]

**Comment by KnightNiwrem** [comment]
Looks all right. Merged.

**Comment by KnightNiwrem** [comment]
Thanks!

**Pull request merged**